xExtension-ImageProxy/extension.php

```diff
@@ -136,7 +136,14 @@ public static function swapUris(string $content): string {
 			}
 		}
 
-		return $doc->saveHTML() ?: '';
+		$body = $doc->getElementsByTagName('body')->item(0);
+
+		$output = $doc->saveHTML($body);
+
+		$output = preg_replace('/^<body>/', '', $output);
+		$output = preg_replace('/<\/body>$/', '', $output);
```

Comment:
```suggestion
		$output = preg_replace('/^<body>|<\/body>$/', '', $output);
```

Not tested, but something like that should work and be faster

Reply:
I would've written it that way, but the example you sent explicitly didn't.